### ISSUE TITLE
Rephrase definition of gcis:derivedModelRunOutputs

### ISSUE THREAD

**Issue by zednis** [USGCRP/gcis-ontology#175]
references #159 

```
gcis:derivedModelRunOutputs a owl:ObjectProperty ;
    rdfs:label "derived model run outputs" ;
    rdfs:comment "A scenario was used in a model run and derived one or more model run outputs." ;
    rdfs:domain gcis:Scenario ;
    rdfs:range gcis:ModelRunOutput ;
    owl:inverseOf gcis:derivedFromScenario ;
    rdfs:subPropertyOf prov:hadDerivation .
```

> @justgo129 
> I don't actually understand the definition. I think "A scenario...derived one or more model outputs outputs" reads somewhat awkwardly to me.

**Comment by zednis** [comment]
how about "outputs of a model run based on this scenario"?

**Comment by zednis** [comment]
@xgmachina @rewolfe

**Comment by justgo129** [comment]
Fine with me.

On Tue, Nov 24, 2015 at 4:58 PM, Stephan Zednik notifications@github.com
wrote:

> how about "outputs of a model run based on this scenario"?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/175#issuecomment-159417549
> .

## 

---

Justin Goldstein, Ph.D.
Advance Science Climate Data and Observing Systems Coordinator
US Global Change Research Program

1800 G Street NW, Suite 9100,  (_Note New Address)_
Washington, D.C. 20006, U.S.A.
phone: (202) 419-3496
e-mail: jgoldstein AT usgcrp Dot gov

http://www.globalchange.gov

**Comment by zednis** [comment]
I'll wait for input from @rewolfe and @xgmachina and if there is consensus make the change.

**Comment by justgo129** [comment]
Sounds good.  In the meantime, for the actual turtle templates, I've merged a pull request incorporating two of the three new predicates.
https://github.com/USGCRP/gcis/pull/250

**Comment by rewolfe** [comment]
+1

**Comment by justgo129** [comment]
Closed #175 due to merged #176.
